src/writer/settings.ts

```diff
@@ -82,6 +82,17 @@ export async function verify(body): Promise<any> {
   const isAdmin = admins.includes(body.address.toLowerCase());
   const newAdmins = (msg.payload.admins || []).map(admin => admin.toLowerCase());
 
+  const anotherSpaceWithDomain = (
+    await db.queryAsync('SELECT 1 FROM spaces WHERE LOWER(domain) = LOWER(?) AND id != ? LIMIT 1', [
+      msg.payload.domain,
+      msg.space
+    ])
+  )[0];
```

Comment:
We should restrict it to be always lowercase here https://github.com/snapshot-labs/snapshot.js/blob/master/src/schemas/space.json#L104-L107

Reply:
See https://github.com/snapshot-labs/snapshot.js/pull/1060